### PR TITLE
new env.list format

### PR DESCRIPTION
**Pull request by michielbdejong**
This avoids a deprecation warning when running `npm run test-api`

### CODE REVIEW

**Comment by vhpoet** [comment]
is this also implemented in the cli?

File: test.env.list [hunk 1]
```diff
@@ -8,9 +8,7 @@ API_MAILGUN_API_KEY=
 API_MAILGUN_DOMAIN=
 API_PORT=3100
 API_PRIVATE_HOSTNAME=localhost
-API_PUBLIC_HTTPS=true
-API_PUBLIC_PATH=/api
-API_PUBLIC_PORT=443
+API_PUBLIC_URI=https://localhost/api
```
Comment (vhpoet) [inline test.env.list]: removed variables are used in `env.js`, please take care of that too.

**Comment by michielbdejong** [comment]
Thanks for your comments, you're right, the change in its current form is too simplistic. I will continue working on this next week.

**Comment by michielbdejong** [comment]
The required changes in `env.js` will remove deprecated behavior, so postponing this to when other breaking changes go into master.

**Comment by michielbdejong** [comment]
Deprecating:
* API_PUBLIC_HTTPS
* LEDGER_PUBLIC_HTTPS
* API_PUBLIC_PATH
* API_PUBLIC_PORT

**Comment by michielbdejong** [comment]
@sharafian can you review this in Vahe's absence?

File: test.env.list (hunk 1 above)
Comment (michielbdejong) [inline test.env.list]: done.

**Comment by michielbdejong** [comment]
I already spent too much time getting this to work and still can't get the tests to pass (although I did learn a lot). Giving up for now, might reopen it in the future, maybe as part of a bigger refactor of the twilight zone.

**Pull request closed without merging**